CHANGELOG.txt

```diff
@@ -181,6 +183,8 @@
                     - Add allowEventOverlaps property (boolean or method).
                     - Add allowEventCreate property (boolean).
         # Javascript:
+            * Deprecations:
+                - Array.copy(iterable, start, end) is deprecated; Use native code Array.from(iterable).slice(start, end) instead.
```

Comment:
There is a section dedicated to deprecations (just below "Non breaking changes")